install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
```

Comment:
You probably want to add this:

```
set -euxo pipefail
```

So the script dies if anything fails unexpectedly.

Reply:
This could be added, let me add it at the top.

Reply:
idk if want to add this - it may just overwhelm the user on his first experience on the install with way too much information such as below. Just `set -e` should be ok, although the script fails with non-zero exit code

```
+ '[' -z /usr/local/bin ']'
+ THE_ARCH_BIN=
+ THIS_PROJECT_OWNER=dunglas
+ THIS_PROJECT_NAME=frankenphp
++ uname -s
+ THISOS=Darwin
++ uname -m
+ ARCH=arm64
```

Reply:
Sounds good to me!